lbz/events/broker.py

```diff
@@ -2,23 +2,26 @@
 from typing import Callable, Dict, List
 
 from lbz.events.event import Event
+from lbz.handlers import BaseHandler
 from lbz.misc import get_logger
 
 logger = get_logger(__name__)
 
 
 # TODO: type_key and data_key will be const for EventBridge and different set for Cognito Events
-class EventBroker:
+class EventBroker(BaseHandler):
```

Comment:
You should remove `pre_handle()` and `post_handle()` from this class as it started using `BaseHandler`.

I don't know why I didn't see that before 🤦